### PR TITLE
Add documentation for users about bigdata and infrastructure occi extension

### PR DESCRIPTION
**Pull request by MaelAudren**

### CODE REVIEW

File: documentation/bigdata_user_guide.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# OCCI Proactive Cloud Automation for BigData
+
+OCCI Proactive Cloud Automation deployment ensure services for deploying spark as a service.
+
+## List the information of all deployed swarm
```
Comment (mbenguig) [inline documentation/bigdata_user_guide.md]: List the informations of all deployed swarm containers

File: documentation/infrastructure_user_guide.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+
+## Describes the information of a deployed compute
+
+```curl -X GET --header 'Accept: application/json' 'http://xx.xxx.xx.xxx:8080/multi-language-connector/occi/compute/urn:uuid:996ad860−2a9a−504f−886−aeafd0b2ae29'```
+
+
+## Create a compute
+
+### Curl complete request :
```
Comment (mbenguig) [inline documentation/infrastructure_user_guide.md]: Full curl request

File: documentation/infrastructure_user_guide.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+curl -X GET --header 'Accept: application/json' 'http://xx.xxx.xx.xxx:8080/multi-language-connector/occi/compute/'
+
+
+## Describes the information of a deployed compute
+
+curl -X GET --header 'Accept: application/json' 'http://xx.xxx.xx.xxx:8080/multi-language-connector/occi/compute/urn:uuid:996ad860−2a9a−504f−886−aeafd0b2ae29'
+
+
+## Create a compute
+
```
Comment (sophiesongge) [inline documentation/infrastructure_user_guide.md]: I think that it's better to explain what is a compute, or use another word.

File: documentation/infrastructure_user_guide.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+},
+  "id": "urn:uuid:996ad860−2a9a−504f−886−aeafd0b2ae29",
+  "kind": "http://schemas.ogf.org/occi/infrastructure#compute",
+  "links": [],
+  "mixins": []
+}' 'http://xx.xxx.xx.xxx:8080/multi-language-connector/occi/compute/'
+</pre>
+</code>
+
+### Curl short request :
```
Comment (mbenguig) [inline documentation/infrastructure_user_guide.md]: Short form curl request

**Pull request merged**